src/diffusers/pipelines/deprecated/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -253,10 +253,14 @@ def __init__(
                 " checker. If you do not want to use the safety checker, you can pass `'safety_checker=None'` instead."
             )
 
-        is_unet_version_less_0_9_0 = hasattr(unet.config, "_diffusers_version") and version.parse(
-            version.parse(unet.config._diffusers_version).base_version
-        ) < version.parse("0.9.0.dev0")
-        is_unet_sample_size_less_64 = hasattr(unet.config, "sample_size") and unet.config.sample_size < 64
+        is_unet_version_less_0_9_0 = is_unet_version_less_0_9_0 = (
```

Comment:
```suggestion
        is_unet_version_less_0_9_0 = (
```
it this intended here?